frontend/src/container/TopNav/DateTimeSelection/index.tsx

```diff
@@ -198,6 +220,63 @@ function DateTimeSelection({
 	};
 
 	// this is triggred when we change the routes and based on that we are changing the default options
+	const getQueryInterval = useCallback(
+		(searchStartTime: string) => {
+			const lastRefresh = dayjs(
+				new Date(parseInt(getTimeString(searchStartTime), 10)),
+			);
+			const { minutedDiff, hoursDiff, daysDiff } = getDiffs(lastRefresh);
+
+			if (daysDiff > 1) {
+				return '1week';
+			}
+
+			if (hoursDiff > 6) {
+				return '1day';
```

Comment:
also i don't think this logic will scale has it is not matching the current available options available for current route

Reply:
actually in service map we don't show the custom